src/Resources/Locales/de_DE.axaml

```diff
@@ -40,13 +40,13 @@
   <x:String x:Key="Text.Archive.Revision" xml:space="preserve">Revision:</x:String>
   <x:String x:Key="Text.Archive.Title" xml:space="preserve">Archiv erstellen</x:String>
   <x:String x:Key="Text.Askpass" xml:space="preserve">SourceGit Askpass</x:String>
-  <x:String x:Key="Text.AssumeUnchanged" xml:space="preserve">UNVERÄNDERTE DATEIEN</x:String>
-  <x:String x:Key="Text.AssumeUnchanged.Empty" xml:space="preserve">KEINE UNVERÄNDERTEN DATEIEN GEFUNDEN</x:String>
+  <x:String x:Key="Text.AssumeUnchanged" xml:space="preserve">ALS UNVERÄNDERT ANGENOMMENE DATEIEN</x:String>
+  <x:String x:Key="Text.AssumeUnchanged.Empty" xml:space="preserve">KEINE UNVERÄNDERT ANGENOMMENEN DATEIEN GEFUNDEN</x:String>
```

Comment:
Yes, but I would add "als": "KEINE **_ALS_** UNVERÄNDERT ANGENOMMENEN DATEIEN GEFUNDEN" (maybe even without "gefunden" because I just saw that is also not part of the original text and was introduced by me).

Reply:
You're right, the "als" is missing there and I'll remove the "gefunden".